backend/src/contaxy/api/endpoints/auth.py

```diff
@@ -87,6 +89,12 @@ def login_user_session(
 
     This will set http-only cookies containg tokens with full user access.
     """
+    system_info = component_manager.get_system_manager().get_system_info()
+    if system_info.system_state != SystemState.RUNNING:
+        raise PermissionDeniedError(
```

Comment:
Maybe it should rather be a 5xx error as it has nothing to do with the user permission per se

Reply:
In case of a 500 error, no useful information is returned to the caller (only "internal server error"). I will then change it to 400, so the caller will still know why the call failed.